apps/web/pages/api/users/username/[username].ts

```diff
@@ -0,0 +1,51 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
This file is using the Pages Router API routes which are deprecated in favor of the App Router. Consider migrating this to a Route Handler in the App Router format, which would be located at `app/api/users/username/[username]/route.ts` instead. Or a trpc endpoint.